operator/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>keycloak-parent</artifactId>
+        <groupId>org.keycloak</groupId>
+        <version>17.0.0-SNAPSHOT</version>
+    </parent>
+    
+    <name>Keycloak Quarkus Operator</name>
+    <artifactId>keycloak-quarkus-operator</artifactId>
+
+    <properties>
+        <!--
+            Override versions based on Quarkus dependencies.
+            Make sure to update these dependencies when Quarkus version changes.
+            See https://github.com/quarkusio/quarkus/blob/<versionTag>/bom/application/pom.xml
+            for reference
+        -->
+        <resteasy.version>4.7.4.Final</resteasy.version>
+        <wildfly.common.version>1.5.4.Final-format-001</wildfly.common.version>
```

Comment:
Do we really need to override it even for the operator?

Reply:
yes, it doesn't even starts otherwise, and we might need more in the future.

Reply:
@andreaTP Could you please mark this as resolved?